ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh

```diff
@@ -107,6 +107,11 @@ fi
 echo "Choosing openshift version ${OPENSHIFT_VERSION}"
 
 # Switches
+TAGS="prowci:${CLUSTER_NAME}"
```

Comment:
Tags should use the format Tags are comma separated, for example: 'key value, foo bar' as the help message shows. It is same to CLUSTER_TAGS. New update.

Reply:
@yasun1 One more thing, if the tags should be included with double quotation marks, or like --tags "a v,fd p:f,l:o pp"